### PR TITLE
CNA/Loigtech - CVE-2022-0915

### PR DESCRIPTION
**Pull request by nhamzi**

### CODE REVIEW

**Comment by cve-team** [comment]
nhamzi is not recognized currently as an account participating in the Git Pilot. To ensure timely processing of your pull request, please contact the CVE Team using either the CVE Request web form (https://cveform.mitre.org/) or an email (cve@mitre.org) and confirm your identity.

File: 2022/0xxx/CVE-2022-0915.json [hunk 1]
```diff
@@ -1,18 +1,100 @@
 {
```
Comment (cve-team) [inline 2022/0xxx/CVE-2022-0915.json:1]: Invalid JSON file (invalid state)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/0xxx/CVE-2022-0915.json (hunk 1 above)
Comment (anthnysingleton) [reply 2022/0xxx/CVE-2022-0915.json:1]: Hello @nhamzi,

The state should be "PUBLIC" and not "REVIEW"

Comment (anthnysingleton) [reply 2022/0xxx/CVE-2022-0915.json:1]: Hello @nhamzi,

The state should be "PUBLIC" and not "REVIEW"

**Pull request merged**